packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -249,6 +249,7 @@ export class ApiReferencesTable extends React.Component<IApiReferencesTableProps
         maxWidth: 400,
         isCollapsible: false,
         isResizable: true,
+        isMultiline: true,
```

Comment:
I don't see this being used anywhere else in this PR - am I missing something?

Reply:
It's allowing enum value descriptions to be displayed as multi-line fields, motivated by the really long (and important) PanelType descriptions.

Today:
<img src="https://user-images.githubusercontent.com/5864305/67898801-3ea37080-fb1e-11e9-88fb-69137aeab86e.png" width=339/>

After this PR: 
<img src="https://user-images.githubusercontent.com/5864305/67898858-55e25e00-fb1e-11e9-998e-691a37c1a239.png" width=407/>

Reply:
Oh I see - should we perhaps have a multiline example then using this?

Reply:
Not sure what you mean? This is just a documentation component, not anything we have examples of.

Reply:
Ahh sorry my bad! Not sure why but yesterday when I was reviewing this, I think it looked like a type declaration to me instead of just the setting.